frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -82,15 +94,17 @@ const FeedbackAlert = () => {
  * @param opts Available feedback options
  * @returns NPSFeedback component
  */
-const NPSFeedback = (opts: FeedbackOptions) => {
+const NPSFeedback = ({ origin = "HEADER", ...options }: FeedbackOptions) => {
   const [hasSubmit, setHasSubmit] = useState<boolean>(false);
-  const [selected, setSelected] = useState<Rating>(null);
+  const [selectedEmoji, setSelectedEmoji] = useState<Rating>(null);
   const [freeformFeedback, setFreeformFeedback] = useState<string>("");
   const [error, setError] = useState<boolean>(false);
   const [survey, setSurvey] = useState<IClutch.feedback.v1.ISurvey>({});
+  const [feedbackType, setFeedbackType] = useState<string>(null);
```

Comment:
something I thought of just now - i'm wondering if there isn't a need to have the `feedbackType` field for the Header & instead standardize the use of `urlPath` for all feedback placement types. Initially I thought there would be value in having both urlPath and feedbackType as data points for the Anytime placement but since we primarily care what workflow received feedback, maybe it's unnecessary to spread the same data across different fields. lmk what you think!

so maybe we can modify like so:
```
const feedbackUrlPath = wizardOrigin ? window.location.pathname : feedbackType
...
},
   urlPath: feedbackUrlPath,
},
```

Reply:
Only issue with this would be if we want to provide feedback outside of workflows (ie General). Since we dont have a true URL for these it would be filled in under urlPath which might seem odd (urlPath: General).

We could possibly swap to just 'feedbackType' and have that be where the url path dumps into.

Reply:
ah yeah true, keeping `feedbackType` would be better if we make this change.

i'll ask the team what they think in standup today. maybe others will think we should keep the 2 fields as planned

Reply:
Sounds good, we could designate '/' for General as well, only might be an issue in future extensions.

Reply:
agree that for future extensions `feedbackType` is more flexible than `urlPath`. let's go with that.

I'll update the mode queries to make sure we're not losing old feedback results where `urlPath` is used

Reply:
Sounds good

Reply:
Merged your changes in!